src/gui/MItem_menus.cpp

```diff
@@ -403,16 +404,26 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
 }
 
 void MI_LOAD_SETTINGS::click(IWindowMenu & /*window_menu*/) {
-    // FIXME: Some error handling/reporting
-    // TODO: Loading other things than just network
-    if (netdev_load_ini_to_eeprom()) {
+    bool network_settings_loaded = netdev_load_ini_to_eeprom();
```

Comment:
Please add all the code using network_settings_loaded close together

Reply:
aye